cmd/agent/subcommands/diagnose/command_test.go

```diff
@@ -115,3 +115,13 @@ func TestShowMetadataSecurityAgentCommand(t *testing.T) {
 			require.Equal(t, false, secretParams.Enabled)
 		})
 }
+
+func TestShowAgentTelemetryCommand(t *testing.T) {
+	fxutil.TestOneShotSubcommand(t,
+		Commands(&command.GlobalParams{}),
+		[]string{"diagnose", "show-metadata", "agent-telemetry"},
+		printPayload,
+		func(_ core.BundleParams, secretParams secrets.Params) {
+			require.Equal(t, false, secretParams.Enabled)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏What is the goal of this test ?

Reply:
I would expect this test to check `"agent-telemetry"`

Reply:
@ogaca-dd @misteriaud fixed 😄